docs/features/CRAWLER_DIRECTIVES.md

```diff
@@ -0,0 +1,252 @@
+Use these tools to manage webpage discovery and content extraction.
+
+**Discovery** concerns which web pages and files from crawled domains get indexed:
+
+- [Canonical URL link tags](#canonical-url-link-tags)
+- [Robots meta tags](#robots-meta-tags)
+- [Nofollow links](#nofollow-links) 
+- [`robots.txt` files](#robotstxt-files) 
+- [Sitemaps](#sitemaps)
+
+**Extraction** concerns how content is indexed and mapped to fields in Elasticsearch documents:
+
+- [Data attributes for inclusion and exclusion](#data-attributes-for-inclusion-and-exclusion) 
+
+## HTML elements and attributes
+
+The following sections describe crawler instructions you can embed within HTML elements and attributes.
+
+### Canonical URL link tags
+
+A canonical URL link tag is an HTML element you can embed within pages that duplicate the content of other pages.
+The canonical URL link tag specifies the canonical URL for that content.
+
+The canonical URL is stored on the document in the url field, while the additional_urls field contains all other URLs where the crawler discovered the same content.
+If your site contains pages that duplicate the content of other pages, use canonical URL link tags to explicitly manage which URL is stored in the url field of the indexed document.
+
+#### Template
+
+```html
+<link rel="canonical" href="{CANONICAL_URL}">
+```
+
+#### Example
+
+```html
+<link rel="canonical" href="https://example.com/categories/dresses/starlet-red-medium">
+```
+
+### Robots meta tags
+
+Robots meta tags are HTML elements you can embed within pages to prevent the crawler from following links or indexing content.
+These tags are related to [crawl rules](./CRAWL_RULES.md).
+
+#### Template
+
+```html
+<meta name="robots" content="{DIRECTIVES}">
+```
+
+#### Supported directives
+
+`noindex`
+
+- The web crawler will not index the page
+- If you want to index some, but not all, content on a page, see [data attributes for inclusion and exclusion](#data-attributes-for-inclusion-and-exclusion)
+
+`nofollow`
+
+- The web crawler will not follow links from the page
+- The directive does not prevent the web crawler from indexing the page
+
+#### Examples
+
+```html
+<meta name="robots" content="noindex">
+<meta name="robots" content="nofollow">
+<meta name="robots" content="noindex, nofollow">
+```
+
+### Data attributes for inclusion and exclusion
+
+Inject HTML data attributes into your web pages to instruct the web crawler to include or exclude particular sections from extracted content.
+For example, use this feature to exclude navigation and footer content when crawling, or to exclude sections of content only intended for screen readers.
+
+These attributes work as follows:
+
+- For all pages that contain HTML tags with a `data-elastic-include` attribute, the crawler will only index content within those tags
+- For all pages that contain HTML tags with a `data-elastic-exclude` attribute, the crawler will skip those tags from content extraction
```

Comment:
```suggestion
- For all pages that contain HTML tags with a `data-elastic-include` attribute, the crawler will only index content within those tags.
- For all pages that contain HTML tags with a `data-elastic-exclude` attribute, the crawler will skip those tags from content extraction.
```

Reply:
Not super important; just making the terminal punctuation consistent within this bulleted list.